openedx/core/djangoapps/profile_images/views.py

```diff
@@ -105,9 +108,24 @@ class ProfileImageView(APIView):
           the user exists or not.
     """
 
-    parser_classes = (MultiPartParser, FormParser)
+    parser_classes = (MultiPartParser, FormParser, TypedFileUploadParser)
     authentication_classes = (OAuth2AuthenticationAllowInactiveUser, SessionAuthenticationAllowInactiveUser)
     permission_classes = (permissions.IsAuthenticated, IsUserInUrl)
+    _supported_media_types = None
+
+    @property
+    def supported_media_types(self):
```

Comment:
Should this helper method belong in `openedx/core/lib/api/parsers.py` instead?  It is not specific to profile_images and other image-related views will want the same functionality.  Perhaps as a default implementation in `TypedFileUploadParser` or as a helper class method?